src/bot/client/ProtomoleculeClient.ts

```diff
@@ -74,11 +90,25 @@ export default class ProtomoleculeClient extends Client {
 			return;
 		}
 
-		// Login to Discord
-		await this.login(this.botToken);
+		if (this.token) {
+			// Login to Discord
+			await this.login(this.token);
 
-		// TODO replace with logger instance
-		// eslint-disable-next-line no-console
-		console.log('Logged in');
+			// TODO replace with logger instance
+			// eslint-disable-next-line no-console
+			console.log('Logged in');
+		} else
+			// TODO replace with logger instance
+			// eslint-disable-next-line no-console
+			console.log('No token present');
+
+		// Load commands
+		try {
+			this.commandManager.init(Path.join(__dirname, '../commands'));
```

Comment:
Do we expect to dynamically configure commands or are they fixed for a particular build of the Protomolecule bot?

Reply:
This was mainly added in preparation for hot-loading/re-loading of commands without having to restart the bot, also with a consideration about future updates.

If we have future contributors, I'd like certain areas to be easy to work with.  This approach just seemed to be the easiest way for anyone to just add a new command without worrying about if it's being loaded or if it's been appropriately registered and initialized.

There very well could be an easier way to accomplish the same task.  I'm open to suggestions.

Reply:
I'm not sure I agree that it's easier for new contributors. Directly importing and using code can be more clear to developers than auto loading code, since things can be pretty opaque if you can't track down the auto loading mechanism. However, if it's a goal that we want to dynamically load plugins, then this seems reasonable.